deno.lock

```diff
@@ -283,13 +296,18 @@
   "npm": {
     "specifiers": {
       "littoral-templates@0.2.2": "littoral-templates@0.2.2",
+      "nanoid@3.3.4": "nanoid@3.3.4",
```

Comment:
I would expect that version 4.0.0 is really _replaced_ by 3.3.4, but like this it's added. (Also below.)

Reply:
Right. I didn't edit the lock file by hand. Maybe there's some other code depending on 4.0.0 that didn't trigger the issue.

Reply:
Running `deno task lock-deps` on the commandline should fix that.